src/hooks/__tests__/useEditProjectSimplified.navigation.test.tsx

```
@@ -65,9 +76,12 @@ describe('useEditProjectSimplified - Navigation Integration', () => {
       },
     });
     vi.clearAllMocks();
+    
+    // Setup navigation mock to resolve successfully
+    mockNavigateToProject.mockResolvedValue({ success: true });
 
     // Mock window.location
-    delete (window as any).location;
+    delete (window as unknown as { location?: Location }).location;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)